CHANGELOG.md

```diff
@@ -8994,7 +8994,7 @@ Swift 1.0
 [SE-0107]: <https://github.com/apple/swift-evolution/blob/main/proposals/0107-unsaferawpointer.md>
 [SE-0108]: <https://github.com/apple/swift-evolution/blob/main/proposals/0108-remove-assoctype-inference.md>
 [SE-0109]: <https://github.com/apple/swift-evolution/blob/main/proposals/0109-remove-boolean.md>
-[SE-0110]: <https://github.com/apple/swift-evolution/blob/main/proposals/0110-distingish-single-tuple-arg.md>
+[SE-0110]: <https://github.com/apple/swift-evolution/blob/main/proposals/0110-distinguish-single-tuple-arg.md>
```

Comment:
This requires renaming the file in the other repository.

Reply:
Please send a PR here https://github.com/apple/swift-evolution/pulls

Reply:
Sorry about the mass closures. I have some tooling that I use, but I accidentally copied the wrong repository name 🤦 , so it deleted all of my existing branches which caused github to close my PRs.